Upgrading versions to support Android Studio Instant Run feature

alrosot opened the pull request:
Android Studio Instant run is quite an useful feature in development environments. But that requires a version bump on gradle and on build-tools

alrosot committed:
build-tools-25.0.2,android-24

Those versions are likely to be used nowadays in development environments

alrosot committed:
Bumping versions (again)

alrosot committed:
Removing broken link

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/uber-common/android-build-environment?pullRequest=14) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/uber-common/android-build-environment?pullRequest=14) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/uber-common/android-build-environment?pullRequest=14) it.</sub>

alrosot committed:
https://github.com/uber-common/android-build-environment/pull/13

alrosot committed:
Installing also Android Support Repository

alrosot committed:
Adding sdk patch applier v4

alrosot committed:
Removing packages already present on ubuntu 14.04
Removing NDK

saschpe commented:
any update on this?

alrosot commented:
I saw that one of my commits was made under a different user (root). Potentially that was preventing my pull request from being merged. I fixed that and updated the pull request.
But the sad true is that this project probably is no longer maintained. (or is super stable)

alrosot committed:
Update README.md
